.github/workflows/coding-standards.yml

```diff
@@ -8,6 +8,20 @@ on:
       - master
       - '3.[89]'
       - '[4-9].[0-9]'
+    paths:
+      # Any change to a PHP or JavaScript file should run checks.
+      - '**.js'
+      - '**.php'
+      # These files configure NPM, which could affect outcome.
+      - 'package*.json'
+      # These files configure Composer, which could affect outcome.
+      - 'composer.*'
+      # Configures JSHint.
+      - '.jshintrc'
+      # Changes to workflow files should always verify all workflows still work.
+      - '.github/workflows/*.yml'
+      # Always run when a version bump occurs.
+      - 'src/wp-includes/version.php'
```

Comment:
1. What's the thinking behind always running the coding standards checks when there's a version bump?
2. This is already covered by the `**.php` path above.

Reply:
My thinking was that the version bump is the last point before release, so it would be good to double check that all workflows pass successfully.

I added it to every workflow just to be explicit. This would also require the release team to wait before building the package. But where build times are now sub 10 minutes. This may not be a big deal. I'm not sure what else could trigger a final a "go/no go" check.